package.json

```diff
@@ -14,9 +14,9 @@
     },
     "repository": {
         "type": "git",
-        "url": "https://github.com/williamspiro/HubL-Language-Extension"
+        "url": "https://github.com/HubSpot/HubL-Language-Extension"
     },
-    "homepage": "https://github.com/williamspiro/HubL-Language-Extension/blob/master/README.md",
+    "homepage": "https://github.com/HubSpot/HubL-Language-Extension/blob/master/README.md",
```

Comment:
I wonder if we should rename the repo. Maybe `hubspot-vscode-extension`, `hubspot-cms-vscode-extension`, or `hubspot-cms-vscode`?

Reply:
I think we should wait to get ownership of the marketplace listing, though